packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -190,3 +190,67 @@ Therefore, the snapshots CI check will fail if the committed Kontrol state diff
 Note that the CI check only compares the JSON state diff, not the generated `DeploymentSummary.sol` or `DeploymentSummaryCode` contracts.
 This is for simplicity, as those three files will be in sync upon successful execution of the `make-summary-deployment.sh` script.
 We commit the `DeploymentSummary.sol` and `DeploymentSummaryCode.sol` contracts, because forge fails to build if those contracts are not present—it is simpler to commit these autogenerated files than to workaround their absence in forge builds.
+
+### KaaS (Kontrol as a Service) CI Integration
+
+KaaS (Kontrol as a Service) is a service provided by Runtime Verification Inc. It is used to execute Kontrol proofs in a CI pipeline. RV offers a hardware cluster to run these proofs, and the results are posted back to the commit statuses for the corresponding run request. GitHub statuses report a pass/fail status back to a commit, and each status result can be viewed by following the "Details" link. In a pull request, this status and link can be found under the PR Checks table at the bottom.
+
+To use KaaS, you need a Runtime Verification assigned token that provides access to run compute jobs under the RV Org. Additionally, if you want to receive status updates on the pass/fail of compute executions, you must provide a Fine-Grained token with the necessary statuses permissions to RV.
+
+A workflow in CircleCI is configured within [.circleci/config.yml](../../../../.circleci/config.yml) and calls on the Github API to execute in the KaaS compute ecosystem. CircleCI must be/is configured with the RV provided Secrets to execute compute jobs in KaaS.
+
+Running `pnpm run test:kontrol` can run a similar environment locally, allowing for reproducing CI tests.
```

Comment:
Can we expand on how to reproduce CI more? e.g. do we need to pass a certain flag to `pnpm test:kontrol`? And what about getting the tokens locally?

Reply:
That is all that has to be done actually. Unless you want to dive into using the run-kontrol.sh script itself to reproduce the CI execution the provided command is that is necessary. I can reference above the documentation related to run-kontrol but that information is well laid out already.

Reply:
Sorry let me clarify: I know `pnpm test:kontrol` is how you run the kontrol tests locally, and it can take args for local/dev/docker mode. But how do you reproduce CI tests that offload compute using `pnpm test:kontrol`? This:

> Running `pnpm run test:kontrol` can run a similar environment locally, allowing for reproducing CI tests.

implies that that it's possible

Reply:
I see. I mentioned this further down in the document which is the 'curl' command used to run it on the remote compute. Requiring the necessary parameters and tokens to be substituted . I can reword this or maybe best to remove this piece all together.

Reply:
Yea I think it's ok to just delete this sentence and rely on the content further down